src/network/MissionControlProtocol.cpp

```diff
@@ -219,9 +221,9 @@ void MissionControlProtocol::sendJointPositionReport(const std::string& jointNam
 	this->_server.sendJSON(Constants::MC_PROTOCOL_NAME, msg);
 }
 
-void MissionControlProtocol::sendCameraStreamReport(const CameraID& cam,
-													const std::string& b64_data) {
-	json msg = {{"type", CAMERA_STREAM_REP_TYPE}, {"camera", cam}, {"data", b64_data}};
+void MissionControlProtocol::sendCameraStreamReport(
+	const CameraID& cam, const std::basic_string<uint8_t>& nal_data) {
+	json msg = {{"type", CAMERA_STREAM_REP_TYPE}, {"camera", cam}, {"data", nal_data}};
```

Comment:
Same comment as before, can we rename `nal_data` to something more descriptive? `frameData`, perhaps?